### PR TITLE
Mention and explain the 'xrefstyle' attribute (#130)

### PR DESCRIPTION
**Pull request by cwickert**
As discussed in #130 in our meeting today, I briefly explained `xrefstyle`.

### CODE REVIEW

File: xml/docu_styleguide.structure.xml [hunk 1]
```diff
@@ -769,10 +769,22 @@ and change the value of &lt;varname&gt;DISPLAYMANAGER&lt;/varname&gt;.</screen>
...
+    </para>
    </example>
-   <para>Keep in mind the following cases where listing cross references
-    is discouraged and must be avoided:</para>
+   <para>
+     This example shows the default display of a cross reference. To change it, use the
+     <tag class="attribute">xrefstyle</tag> attribute, either with
+     <tag class="attvalue">select:</tag> or <tag class="attvalue">template:</tag>. Do not use custom
+     <quote>named</quote> <tag class="attribute">xrefstyle</tag> attributes that require support
+     from the stylesheets. For more info, refer to <link
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:780]:
```suggestion
     from the style sheets. For more info, refer to <link
```

**Review by dariavladykina — APPROVED** [review]
I just corrected "style sheets", otherwise LGTM. Thanks!

**Review by jfaltenbacher — APPROVED** [review]
LGTM. Thank you Christoph

File: xml/docu_styleguide.structure.xml [hunk 2]
```diff
@@ -769,10 +769,22 @@ and change the value of &lt;varname&gt;DISPLAYMANAGER&lt;/varname&gt;.</screen>
    <example xml:id="ex-cross-reference-output">
     <title>Example of a cross reference (output)</title>
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:770]:
```suggestion
    <title>Example of a cross-reference (output)</title>
```

File: xml/docu_styleguide.structure.xml [hunk 3]
```diff
@@ -769,10 +769,22 @@ and change the value of &lt;varname&gt;DISPLAYMANAGER&lt;/varname&gt;.</screen>
...
     <para>
-     See <xref linkend="sec-cross-reference"/>.</para>
+     See <xref linkend="sec-cross-reference"/>.
+    </para>
    </example>
-   <para>Keep in mind the following cases where listing cross references
-    is discouraged and must be avoided:</para>
+   <para>
+     This example shows the default display of a cross reference. To change it, use the
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:776]:
```suggestion
     This example shows the default display of a cross-reference. To change it, use the
```

File: xml/docu_styleguide.structure.xml [hunk 4]
```diff
@@ -769,10 +769,22 @@ and change the value of &lt;varname&gt;DISPLAYMANAGER&lt;/varname&gt;.</screen>
...
+     <tag class="attribute">xrefstyle</tag> attribute, either with
+     <tag class="attvalue">select:</tag> or <tag class="attvalue">template:</tag>. Do not use custom
+     <quote>named</quote> <tag class="attribute">xrefstyle</tag> attributes that require support
+     from the style sheets. For more info, refer to <link
+       xlink:href="http://www.sagehill.net/docbookxsl/CustomXrefs.html">DocBook XSL: The Complete
+       Guide: Customizing cross references</link>.
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:782]:
```suggestion
       Guide: Customizing cross-references</link>.
```

File: xml/docu_styleguide.structure.xml [hunk 5]
```diff
@@ -769,10 +769,22 @@ and change the value of &lt;varname&gt;DISPLAYMANAGER&lt;/varname&gt;.</screen>
...
+   </para>
+   <para>
+     Keep in mind the following cases where listing cross references is discouraged and must be
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:785]:
```suggestion
     Keep in mind the following cases where listing cross-references is discouraged and must be
```

**Pull request merged**